cmd/spire-agent/cli/run/run.go

```diff
@@ -46,18 +48,20 @@ type Config struct {
 }
 
 type agentConfig struct {
-	DataDir           string `hcl:"data_dir"`
-	EnableSDS         bool   `hcl:"enable_sds"`
-	InsecureBootstrap bool   `hcl:"insecure_bootstrap"`
-	JoinToken         string `hcl:"join_token"`
-	LogFile           string `hcl:"log_file"`
-	LogFormat         string `hcl:"log_format"`
-	LogLevel          string `hcl:"log_level"`
-	ServerAddress     string `hcl:"server_address"`
-	ServerPort        int    `hcl:"server_port"`
-	SocketPath        string `hcl:"socket_path"`
-	TrustBundlePath   string `hcl:"trust_bundle_path"`
-	TrustDomain       string `hcl:"trust_domain"`
+	DataDir              string `hcl:"data_dir"`
+	EnableSDS            bool   `hcl:"enable_sds"`
+	InsecureBootstrap    bool   `hcl:"insecure_bootstrap"`
+	JoinToken            string `hcl:"join_token"`
+	LogFile              string `hcl:"log_file"`
+	LogFormat            string `hcl:"log_format"`
+	LogLevel             string `hcl:"log_level"`
+	SDSDefaultSVIDName   string `hcl:"sds_default_svid_name"`
+	SDSDefaultBundleName string `hcl:"sds_default_bundle_name"`
```

Comment:
I wonder how folks would feel about grouping these options into an `sds` subsection of the config?

```
agent {
    ...
    sds {
        default_svid_name = "whatever"
    }
}

Reply:
>I wonder how folks would feel about grouping these options into an sds subsection of the config?

I think that grouping the sds specific settings would be good to have a better organization of the config settings, also considering that we may need to add more sds settings in the future.

Reply:
Yeah I agree relatively strongly we should group SDS settings